docs/spectrum/index.rst

```diff
@@ -10,110 +10,116 @@ Spectrum estimation and modeling (`gammapy.spectrum`)
 
 Introduction
 ============
+`gammapy.spectrum` holds functions and classes related to 1D region based spectral analysis.
+The basic of this type of analysis are explained in `this <https://github.com/gammapy/PyGamma15/tree/gh-pages/talks/analysis-classical>`__ talk
 
-`gammapy.spectrum` holds functions and classes to fit spectral models and compute flux points.
-
-Physical radiative models (synchrotron, inverse Compton and pion-decay emission)
-for arbitrary cosmic ray particle spectra are available in the `naima`_ package.
-
-Explain spectrum estimation basics.
-
-Define vocabulary.
+TODO: explain basics
 
 A good reference for the forward-folding on-off likelihood fitting methods is Section 7.5 "Spectra and Light Curves" in [Naurois2012]_,
 in publications usually the reference [Piron2001]_ is used.
 A standard reference for the unfolding method is [Albert2007]_.
 
-.. _spectrum_getting_started:
 
-Getting Started
-===============
+Spectral Fitting
+================
 
-Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool. 
+.. _spectrum_command_line_tool:
 
-The spectral fitting command-line tool makes use of the data management functionality in Gammapy. In order to download an example dataset from the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__ repository and set up an example `gammapy.obs.DataManager` please follow the instructions in :ref:`obs_dm`. The following step assume you have this example data set. If you already have a data set, please modify the steps below accordingly.
+Command line tool
+-----------------
+
+Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool.
+The example below shows how to use ``gammapy-spectrum`` by specifying analysis
+options in a YAML config file. It assumes you have the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__
+repository available.
 
-This is an example config file (YAML format) to be used with the ``gammapy-spectrum`` command line tool.
 
 .. include:: ./analysis_example.yaml
     :code: yaml
 
 
-Copy it to for example ``crab_config.yaml`` and run
+Copy the above config file to your machine, to e.g. ``crab_config.yaml`` and run
 
 .. code-block:: bash
 
    gammapy-spectrum crab_config.yaml
 
-.. _energy_handling_gammapy:
 
-Energy handling in Gammapy
-==========================
+Underlying classes
+------------------
 
-Basics
-------
+The spectral fitting procedure is a two step process. Each of the two steps is represented by one class.
 
-Most objects in Astronomy require an energy axis, e.g. counts spectra or
-effective area tables. In general, this axis can be defined in two ways.
+* The `~gammapy.spectrum.SpectrumAnalysis` class converts the data from the
+  Fermi-LAT format (add link) into the OGIP format
```

Comment:
Can you add a link to the new format spec repo now?

We could do it later, or I could do it, or you could use it as an exercise in how [intersphinx](http://sphinx-doc.org/latest/ext/intersphinx.html) works.

First you have to add a label here and merge that in master of that repo so that the label exists in the intersphinx inventory on readthedocs:
https://github.com/gammapy/gamma-astro-data-formats/blame/master/source/irfs/index.rst#L2

Example:
https://github.com/gammapy/gamma-astro-data-formats/blame/master/source/irfs/background/index.rst#L3

Then you can reference it from the Gammapy docs via
```
:ref:`gadf:label_name`
```
Example: https://github.com/gammapy/gammapy/pull/394/files#diff-d1a23d226cf73b27e37b9996592c9fe4R9

This works because `gadf` is defined as a reference to the spec repo here:
https://github.com/gammapy/gammapy/blob/master/docs/conf.py#L66